src/components/landing-page/landing-page.js

```diff
@@ -1,15 +1,17 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React from "react";
+import { Link } from "react-router-dom";
 
 class LandingPage extends React.Component {
-    render() {
-        return (
-            <div>
-                <h1>Welcome!</h1>
-                <Link to='/preferences'>Click Me</Link>
-            </div>
-        )
-    }
+  render() {
+    return (
+      <div>
+        <h1>Welcome!</h1>
+        <Link to="/preferences">User Form (Ben)</Link>
+        <div />
+        <Link to="/restaurant-details">Restaurant Details (Alexis)</Link>
```

Comment:
in the future can use a button as link
https://stackoverflow.com/questions/42463263/wrapping-a-react-router-link-in-an-html-button#answer-46088718

Reply:
okk yeah this is just placeholder for now for them to toggle between their stuff

Reply:
yeah i just wanted to save the stack overflow answer here haha